do-app-platform-op-cli/README.md

```diff
@@ -1,314 +1,277 @@
-```
+The pod for 1Password SCIM Bridge should be vertically scaled if you provision a large number of users or groups. These are our default resource specifications and recommended configurations for provisioning at scale:
 
-Using Powershell: 
-```pwsh
-Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | doctl apps propose --spec -
-```
-
-### Resource recommendations
+| Volume  | Number of users | CPU  | memory |
+| ------- | --------------- | ---- | ------ |
+| Default | <1,000          | 0.25 | 0.5Gi  |
+| High    | >1,000          | 1.0  | 1.0Gi  |
 
-The 1Password SCIM Bridge Pod should be vertically scaled when provisioning a large number of users or groups. Our default resource specifications and recommended configurations for provisioning at scale are listed in the below table:
+If you're provisioning more than 1,000 users, update the resources assigned to the SCIM bridge container to follow these recommendations. The resources specified for the Redis container don't need to be adjusted.
 
-| Volume    | Number of users | CPU   | memory |
-| --------- | --------------- | ----- | ------ |
-| Default   | <1,000          | 0.25  | 0.5Gi  |
-| High      | 1,000–5,000     | 0.5   | 1.0Gi  |
-| Very high | >5,000          | 1.0   | 1.0Gi  |
+#### Default deployment
 
-If provisioning more than 1,000 users, the resources assigned to the SCIM bridge container should be updated as recommended in the above table. The resources specified for the Redis container do not need to be adjusted.
+If you're provisioning up to 1,000 users, run the following command:
 
-Our current default resource requirements are:
+**Bash**:
 
-<details>
-  <summary>Default</summary>
-
-To revert to the default specification that is suitable for provisioning up to 1,000 users:
-
-Using Bash:
 ```sh
 curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
 ```
 
-Using Powershell:
+**Powershell**:
+
 ```pwsh
 Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
 ```
-</details>
 
-These values can be scaled down again to the default values after the initial large provisioning event.
+#### High-volume deployment
 
-<details>
-  <summary>High volume deployment</summary>
+If you're provisioning more than 1,000 users:
+
+1. Download the [`op-scim-bridge.yaml`](/do-app-platform-op-cli/op-scim-bridge.yaml) file and open it in a text editor.
+2. Update line 37 to the following:
+	```
+	instance_size_slug: basic-xs
+	```
+3. Copy and paste the following command, replace `<pathToFile>` with the path to the file you just edited, and run the command.
+	```sh
+	cat <pathToFile>/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
+	```
```

Comment:
A few fixes/suggestions for this bit:

- some white space fixes
  - empty line before and after code blocks
  - spaces versus tab stops
- added language (YAML) to code block
- prefer a real file path that is likely to work as typed rather than a placeholder
- prefer pointing to relevant code rather than quoting a line number that may change

```suggestion
2. Change the `instance_size_slug` value for the `op-scim-bridge` service to `basic-xs`:

    ```yaml
    services:
    # ...
    - ### ...
      instance_size_slug: basic-xs
      name: op-scim-bridge
    ```

3. Run the following command (replace `./op-scim-bridge.yaml` with the path to your modified app spec file as needed):

    ```sh
    cat ./op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
    ```
```